src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep2.java

```diff
@@ -123,11 +132,23 @@ public void onPrimary() {
 		modalPresenter.setLoading(true);
 		accessRequirement.setAreOtherAttachmentsRequired(view.areOtherAttachmentsRequired());
 		String expirationPeriod = view.getExpirationPeriod();
+		String renewalDetailsURL = view.getRenewalDetailsURL();
+		
+		if (!ValidationUtils.isValidUrl(renewalDetailsURL, true)) {
+			modalPresenter.setErrorMessage(INVALID_RENEWAL_URL_MESSAGE);
+			return;
+		}
+		if (renewalDetailsURL != null && renewalDetailsURL.trim().isEmpty()) {
+			// service will reject if setting to the empty string with an undefined expiration period.
+			renewalDetailsURL = null;
+		}
+		accessRequirement.setRenewalDetailsUrl(renewalDetailsURL);
+		
 		if (DisplayUtils.isDefined(expirationPeriod)) {
 			try {
 				long expirationPeriodInDays = Long.parseLong(expirationPeriod);
-				if (expirationPeriodInDays < 0) {
-					throw new NumberFormatException("Must be a positive integer.");
+				if (expirationPeriodInDays < 0 || (expirationPeriodInDays > 0 && expirationPeriodInDays < 365)) {
+					throw new NumberFormatException("If expiration period is set, then it must be a greater than a year.");
```

Comment:
If expiration period is set, then it must be a greater than a year -> If expiration period is set, then it must be greater than a year